src/editor/components/gridItem.ts

```diff
@@ -124,23 +133,28 @@ export class GridItem extends Panel {
 
     const content = document.createElement('div');
     content.className = 'grid-stack-item-content';
+
+    const button = document.createElement('div');
+    button.className = 'close-button';
+    const close = document.createElement('div');
```

Comment:
We might be able to use a `LabIcon` here, to make it easier to handle themes: https://jupyterlab.readthedocs.io/en/latest/developer/ui_components.html#labicon-set-up-and-render-icons

The icon could even be the same as the one used for the `Delete` entry in the file browser context menu:

![image](https://user-images.githubusercontent.com/591645/97324393-130eaf80-1872-11eb-8d14-854055e01389.png)

Reply:
Done in https://github.com/hbcarlos/voila-editor/pull/36/commits/ac90960094f661a097de0ea3c21401910d231524

![trash-labicon](https://user-images.githubusercontent.com/591645/97421039-54539d80-190c-11eb-8c59-a92d4c7b24d8.gif)